logic/src/main/java/nl/tehdreamteam/se42/logic/exception/impl/DefaultExceptionHandler.java

```diff
@@ -0,0 +1,16 @@
+package nl.tehdreamteam.se42.logic.exception.impl;
+
+import nl.tehdreamteam.se42.logic.exception.ExceptionHandler;
+import nl.tehdreamteam.se42.logic.exception.ServerError;
+
+/**
+ * Default implementation of {@code ExceptionHandler}.
+ *
+ * @author Oscar de Leeuw
+ */
+public class DefaultExceptionHandler implements ExceptionHandler {
+    @Override
+    public ServerError handle(Exception e) {
+        return new ServerError(e, "Not supported yet", 500);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 500 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS109)